### ISSUE TITLE
load dxf file failed

### ISSUE THREAD

**Issue by lainegates** [assimp/assimp#3395]
**Describe the bug**
The Assimp crash when loading .dxf file.
Crashed at 
![image](https://user-images.githubusercontent.com/5265648/92984610-04e11b80-f4de-11ea-88f8-bc8654f26522.png)

crashed at code 
```
LineSplitter& LineSplitter::operator++() {
    if (mSwallow) {
        mSwallow = false;
        return *this;
    }

    if (!*this) {
        throw std::logic_error("End of file, no more lines to be retrieved."); // crashed at line 173 !!!
    }
```

[test.zip](https://github.com/assimp/assimp/files/5212036/test.zip)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 5.0.1
And compile failed with default *master* branch

**Comment by n4pt1me** [comment]
I have the same problem, it would be great to have this fixed. Same OS and assimp version.

**Comment by Optimizer0** [comment]
I tried with the latest master branch code. The model attached here loaded fine and did not give any errors.

However I couldn't see the imported mesh in my OpenGL application no matter how much I scaled up or down BUT the same happened when I tried importing the file into Blender 2.81a using "Import AutoCAD DXF Format (.dxf)" addon latest version (0.9.6). 

It could be that both the assimp importer and blender addon is faulty or maybe the file is faulty. I do not have AutoCAD to test.
